clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkClientTest.java

```diff
@@ -69,6 +71,36 @@ public void send() {
         assertEquals(Errors.NONE, response.error());
     }
 
+    @Test
+    public void sendWithinBlackoutPeriodAfterAuthenticationFailure() throws InterruptedException {
+        client.authenticationFailed(node, 300);
+        client.prepareResponse(heartbeatResponse(Errors.NONE));
+        final RequestFuture<ClientResponse> future = consumerClient.send(node, heartbeat());
+        consumerClient.poll(future);
+        TestUtils.waitForCondition(new TestCondition() {
```

Comment:
I think this test condition is unnecessary. The `poll` in the line above completes only when the future is done. Same below.

Reply:
You're right. Thanks for catching the redundancy. Will remove them in the new commit.